.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: API tests
+
+on:
+  push:
```

Comment:
Should we have this running on just PRs to main ? Could be more efficient imo

`pull_request:
    branches:
      - main`